From 76c617b7a4b11d6d8abaae843f2daa6aa8924b42 Mon Sep 17 00:00:00 2001
From: Jeremy Bicha <jbicha@ubuntu.com>
Date: Tue, 7 Feb 2017 15:21:20 -0500
Subject: [PATCH 1/4] data: Update gnome-session file for gnome-settings-daemon
 3.23

gnome-settings-daemon has been split up into separate daemons, which
means we'll need to invoke those separately.

See https://git.gnome.org/browse/gnome-session/commit?id=18b6e567e1a
for the corresponding change to the regular session file.

https://bugzilla.gnome.org/show_bug.cgi?id=778309
---
 data/gnome-initial-setup.session | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/gnome-initial-setup.session b/data/gnome-initial-setup.session
index b24da3d..0d81bb1 100644
--- a/data/gnome-initial-setup.session
+++ b/data/gnome-initial-setup.session
@@ -1,3 +1,3 @@
 [GNOME Session]
 Name=GNOME Initial Setup
-RequiredComponents=setup-shell;gnome-initial-setup;gnome-settings-daemon;
+RequiredComponents=setup-shell;gnome-initial-setup;org.gnome.SettingsDaemon.A11yKeyboard;org.gnome.SettingsDaemon.A11ySettings;org.gnome.SettingsDaemon.Clipboard;org.gnome.SettingsDaemon.Color;org.gnome.SettingsDaemon.Datetime;org.gnome.SettingsDaemon.Housekeeping;org.gnome.SettingsDaemon.Keyboard;org.gnome.SettingsDaemon.MediaKeys;org.gnome.SettingsDaemon.Mouse;org.gnome.SettingsDaemon.Orientation;org.gnome.SettingsDaemon.Power;org.gnome.SettingsDaemon.PrintNotifications;org.gnome.SettingsDaemon.Rfkill;org.gnome.SettingsDaemon.ScreensaverProxy;org.gnome.SettingsDaemon.Sharing;org.gnome.SettingsDaemon.Smartcard;org.gnome.SettingsDaemon.Sound;org.gnome.SettingsDaemon.Wacom;org.gnome.SettingsDaemon.XRANDR;org.gnome.SettingsDaemon.XSettings;
-- 
2.13.6

